docs/admin-guide/containers/container-deploy.md

```diff
@@ -35,11 +35,11 @@ docker run --net=host --rm --expose 5000 -p 5000:5000 clinicalgenomics/scout sco
 
 From a Mac machine the same command would be slightly different (the reason is described [here](https://docs.docker.com/desktop/mac/networking/)):
 ```
-docker run --rm --expose 5000 -p 5000:5000 clinicalgenomics/scout scout --host docker.for.mac.localhost -db scout-demo  serve --host 0.0.0.0
+docker run --platform=linux/amd64 --rm --expose 5000 -p 5000:5000 clinicalgenomics/scout scout --host docker.for.mac.localhost -db scout-demo  serve --host 0.0.0.0
```

Comment:
Yep! I still set `export DOCKER_DEFAULT_PLATFORM=linux/amd64` in a startup file not to forget it, but this should age better.